src/vm.js

```diff
@@ -921,6 +921,45 @@ class VM extends common.ServiceObject {
       callback || common.util.noop
     );
   }
+  /**
+   * Suspend the instance.
+   *
+   * @see [Instances: Suspend API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instances/suspend}
```

Comment:
this is probably the line that is making our docs tests angry. I think you could just add this link to the linkinator file to ignore for the time being.